c2cgeoform/templates/navigation.jinja2

```diff
@@ -15,7 +15,7 @@
       <ul class="nav navbar-nav">
         {% for table in tables %}
         <li role="presentation" class="{{'active' if request.matchdict and request.matchdict.get('table') == table['key'] else ''}}">
-          <a href="{{request.route_url('c2cgeoform_index', table=table['key'])}}">{{table['plural']}}</a>
+          <a href="{{request.route_url('c2cgeoform_index', table=table['key'])}}">{{_(table['plural'])}}</a>
```

Comment:
plural is not already localized?

Reply:
it is already a TranslationString object, but for the moment I do not really understand why this is not translated without _ call.